tools/SetupFlow/DevHome.SetupFlow/Services/ConfigurationFileBuilder.cs

```diff
@@ -208,14 +212,15 @@ private WinGetConfigResource CreateResourceFromTaskForWinGetDsc(InstallPackageTa
     /// <returns>The WinGetConfigResource object that represents the block of yaml needed by GitDsc to clone the repository. </returns>
     private WinGetConfigResource CreateResourceFromTaskForGitDsc(CloneRepoTask task, Uri webAddress, ConfigurationFileKind configurationFileKind)
     {
-        // WinGet configure uses the Id property to uniquely identify a resource and also to display the resource status in the UI.
-        // So we add a description to the Id to make it more readable in the UI. These do not need to be localized.
-        var id = $"Clone {task.RepositoryName}: {task.CloneLocation.FullName}";
+        // WinGet configure uses the Id property to uniquely identify a resource and also to display the resource status in the UI.
+        // So we add a description to the Id to make it more readable in the UI. These do not need to be localized.
+        var id = $"{RepoNamePrefix}{task.RepositoryName}{RepoNameSuffix}{task.CloneLocation.FullName}";
+
         var gitDependsOnId = DscHelpers.GitWinGetPackageId;
 
         if (configurationFileKind == ConfigurationFileKind.SetupTarget)
-        {
-            gitDependsOnId = $"{DscHelpers.GitWinGetPackageId} | Install: {DscHelpers.GitName}";
+        {
+            gitDependsOnId = $"{DscHelpers.GitWinGetPackageId}{PackageNameSeparator}{DscHelpers.GitName}";
```

Comment:
I changed this to depend on: `Git.GIt`. Please ensure that the `id` for the Git WinGet package and the depends-on `id` are aligned.

Reply:
This PR is already merged with your [Initial support for generating a configuration file from machine configuration](#2466) change. The result of this line does not change here, just replacing "| Install:" with a constant.
However, without the change on line 246 cloning a repo was broken.